.idea/libraries/com_bluecatcode_junit_junit_4_10_extended_1_0_4.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
`.idea` should be in `.gitignore`

some tuts I've found:
 - https://www.youtube.com/watch?v=mf2-MOl0VXY
 - https://www.youtube.com/watch?v=gKWnufyBNQA
 - https://www.youtube.com/watch?v=6vvUevxlOp8

Reply:
added to .gitignore